src/Native/gc/gcscan.h

```diff
@@ -15,6 +15,7 @@
 #define _GCSCAN_H_
 
 #include "gc.h"
+#include "volatile.h"
```

Comment:
The idea was that `gcenv.h` contains all external header files required by the GC. It would be better to add this to gcenv.h instead.

Reply:
Ah gotcha, i was confused because `volatile.h` got vendored into the GC but then was missing on build, i take it, this ensures the runtime version of `Volatile.h` gets used when not building the standalone gc.